internal/plugin/resource_vault_start.go

```diff
@@ -325,6 +382,34 @@ As such, you will need to provide _all_ values except for ^seals^ until we make
 - ^config.seals.tertiary.type^ (String) The Vault [seal](https://developer.hashicorp.com/vault/docs/configuration/seal) type
 - ^config.seals.tertiary.attributes^ (String) The Vault [seal](https://developer.hashicorp.com/vault/docs/configuration/seal) parameters for the given seal type
 - ^config.telemetry^ (Object) The Vault [telemetry](https://developer.hashicorp.com/vault/docs/configuration/telemetry#telemetry-parameters) stanza
+- ^config.ha_storage^ (Object) The Vault [ha_storage](https://developer.hashicorp.com/vault/docs/internals/high-availability?productSlug=vault&tutorialSlug=raft&tutorialSlug=raft-ha-storage) stanza
```

Comment:
We'll need to run [make docs](https://github.com/hashicorp-forge/terraform-provider-enos/blob/main/Makefile#L46) before merge to ensure our rendered docs are accurate.